public/angular/templates/email_modal.html

```diff
@@ -0,0 +1,45 @@
+<div>
+  <div class="modal-header">
+    <button class="close" ng-click="close()" aria-hidden="true">×</button>
+    <h4 class="modal-title">
+      <div class="transport">
+        <div>Edit Email Transport Settings</div>
+    </h4>
+    </div>
+    <div class="modal-body">
+      <div class="well bs-component">
+        <form name="form" class="form-horizontal">
+          <fieldset>
+            <div class="form-group">
+              <label for="transport" class="col-lg-2 control-label">Transport</label>
+              <div class="col-lg-8">
+                <select name="method" class="form-control" ng-model="transport.method"
+                          ng-options="name as name for (name, settings) in emailTransportOptions">
+                  <option value="">Choose transport method</option>
+                </select>
+                <div class="has-error" ng-show="showErrors()">
+                  <div class="help-block" ng-show="form.method.$error.required">Select transport first</div>
+                </div>
+              </div>
+            </div>
+
+            <div ng-repeat="setting in emailTransportOptions[transport.method]">
+              <label for="{{ setting }}" class="col-lg-2 control-label">{{ setting }}</label>
```

Comment:
I'd like to have more human friendly labels for these fields. The best way to do this would be to use the i18n stuff, but it's not ready, so I'll put an issue in the backlog for now.